<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettings</class>
 <widget class="QDialog" name="CameraSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Camera Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="brightness" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Brightness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="brightness_sld">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="brightness_sb">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contrast" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Contrast</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="contrast_sld">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="contrast_sb">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="gain" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Gain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="gain_sld">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="gain_sb">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="saturation" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Saturation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="saturation_sld">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="saturation_sb">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="hue" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Hue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hue_sld">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="hue_sb">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="gamma" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Gamma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="gamma_sld">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="gamma_sb">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="sharpness" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Sharpness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sharpness_sld">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sharpness_sb">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="auto_exposure" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Auto Exposure</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="expo_cb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="map_file" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Map File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="map_file_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="open_file_pb">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>238</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>brightness_sld</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brightness_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightness_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brightness_sld</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrast_sld</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrast_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrast_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrast_sld</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturation_sld</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturation_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturation_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturation_sld</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sharpness_sld</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sharpness_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sharpness_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sharpness_sld</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gain_sld</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gain_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gain_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gain_sld</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hue_sld</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hue_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hue_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hue_sld</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
